src/components/Api/index.js

```diff
@@ -0,0 +1,3 @@
+import Api from './Api';
```

Comment:
optional: if you move your entire Api.js file into index, then you don't need this file at all

Reply:
So the only file in src/components/Api would be index.js?

Reply:
yes